libs/domain/picking/src/components/customer-note/src/customer-note.component.ts

```diff
@@ -41,8 +41,14 @@ export class CustomerNoteComponent extends PickingListMixin(LitElement) {
 
       <p>${this.pickingList?.cartNote}</p>
 
-      <oryx-button>
-        <button @click=${this.onProceed}>
+      <oryx-button
+        ?loading=${this.upcomingPickingListId === this.pickingList.id}
```

Comment:
For this case is enough just to check whether `upcomingPickingListId` truthy  or not. There are no another picking lists, and this changes can be initiated by this component itself only

`?loading=${!!this.upcomingPickingListId}`